### PR TITLE
Add DeepVariant tool

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/deepvariant/deepvariant.xml [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+        --call_variants_extra_args="use_openvino=true" ## Setting this will use OpenVINO on Intel CPUs, which empirically reduces call_variants runtime by 15%-25%.
+        --num_shards=\${GALAXY_SLOTS:-2}
+        && gunzip './output.vcf.gz'
+        #if $output_gvcf
+            && gunzip './output.g.vcf.gz'
+        #end if
+    ]]>    </command>
+    <inputs>
+        <conditional name="reference_genome">
+            <param name="source" type="select" label="Source for the reference genome" help="Built-in references were created using default options.">
```
Comment (bgruening) [inline tools/deepvariant/deepvariant.xml:43]: Should we add here that this tool only works reliably on human genomes?

Comment (gallardoalba) [reply tools/deepvariant/deepvariant.xml:43]: In theory is possible to use it in other organisms, but in that case perhaphs is better to restrict it to Humans? 

This is what the documentation says:
"The models included with DeepVariant are only trained on human data. For other organisms, see the blog post on non-human variant-calling for some possible pitfalls and how to handle them."

Comment (bgruening) [reply tools/deepvariant/deepvariant.xml:43]: I think if we add this sentence somewhere this is fine.

Comment (gallardoalba) [reply tools/deepvariant/deepvariant.xml:43]: It was included before in the help section, but I didn't realize about it.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Awesome, thanks @gallardoalba!

**Comment by mvdbeek** [comment]
@bgruening I think we should have had a discussion first on whether we will merge container-only tools and under which conditions we will do that, and whether there need to be improvements in Galaxy and the TS interface on how admins are alerted to this fact. There is https://anaconda.org/bioconda/deepvariant, was there a reason this couldn't be used ?

**Comment by gallardoalba** [comment]
> @bgruening I think we should have had a discussion first on whether we will merge container-only tools and under which conditions we will do that, and whether there need to be improvements in Galaxy and the TS interface on how admins are alerted to this fact. There is https://anaconda.org/bioconda/deepvariant, was there a reason this couldn't be used ?

I just used it because it is the developer's recommendation https://github.com/google/deepvariant#how-to-run-deepvariant

**Comment by bgruening** [comment]
> There is https://anaconda.org/bioconda/deepvariant, was there a reason this couldn't be used?

This package has multiple problems and no tests: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/deepvariant/meta.yaml
The real show-stopper is that it does not compile under older glibc and thus does not fulfill the usual requirements.

**Pull request merged**